### DISCUSSION TITLE
More information like active resolution in the debug overlay menu

### DISCUSSION THREAD

**Discussion by Psybertr0n** [f1multiviewer/issue-tracker#43] [Feature requests] [answered]
In the overlay revealed when we press D can we also have the active video resolution displayed? That way we can know which resolution Auto has selected for us or we can confirm that our chosen resolution is indeed active. 

I've shown the info overlay of mpv player in the screenshot below as an example. I can see that it's a 1080p native stream which is scaled to 2160p on my screen.

![mpv_2022-06-17_22-19-24](https://user-images.githubusercontent.com/16277629/174343858-61e93fbe-8692-4e7d-b8f2-7885497e2ab7.jpg)

It would be great to also have these things displayed:
- video bitrate
- audio sample rate 
- audio bitrate
- dropped frames

**Comment by f1multiviewer** [comment, marked as answer]
Done (a few releases ago)
